# HG changeset patch
# User Rik <rik@octave.org>
# Date 1515563981 28800
#      Tue Jan 09 21:59:41 2018 -0800
# Node ID 304501c692e66bf7af2b03d9d07e630db9293ff0
# Parent  3ee61988e9a9ec03a9eb23d80b71185f72cdfeca
bounds.m: Fix failing BIST tests.

* bounds.m: Fix failing BIST tests.

diff --git a/scripts/statistics/bounds.m b/scripts/statistics/bounds.m
--- a/scripts/statistics/bounds.m
+++ b/scripts/statistics/bounds.m
@@ -80,19 +80,25 @@ function [s, l] = bounds (x, dim, nanfla
     nanidx = any (isnan (x), dim); 
     s(nanidx) = NaN; 
     l(nanidx) = NaN; 
   endif
 
 endfunction
 
 
-%!assert (bounds (1:10), [1, 10])
-%!assert (bounds ([10:-1:1]'), [1, 10])
-%!assert (bounds (single (1:10)), single ([1, 10]))
+%!test
+%! [s,l] = bounds (1:10);
+%! assert ([s,l], [1, 10]);
+%!test
+%! [s,l] = bounds ([10:-1:1]');
+%! assert ([s,l], [1, 10]);
+%!test
+%! [s,l] = bounds (single (1:10));
+%! assert ([s,l], single ([1, 10]));
 %!assert (bounds (magic (3)), [3, 1, 2])
 %!assert (bounds (magic (3), 2), [1; 3; 2])
 %!test
 %! x = magic (3);
 %! x(2,3) = NaN;
 %! assert (bounds (x), [3, 1, 2]);
 %! assert (bounds (x, "omitnan"), [3, 1, 2]);
 %! assert (bounds (x, "includenan"), [3, 1, NaN]);
